plugins/woocommerce/tests/php/includes/emails/class-wc-email-customer-pos-completed-order-test.php

```diff
@@ -28,7 +28,7 @@ public function setUp(): void {
 	public function test_order_item_totals_adds_formatted_cash_change_due_amount_to_order_totals() {
 		// Given order with cash payment change amount.
 		$order = OrderHelper::create_order();
-		$order->add_meta_data( '_cash_change_amount', '5.00' );
+		$order->add_meta_data( '_cash_change_amount', '0' );
```

Comment:
Why change the value in this test? Perhaps it would be worth having a separate test for 0 change due?

Reply:
Most to reduce the number of diffs by keeping just the edge case. The PR is still in manageable size, so I updated for the zero amount case to be a separate case in https://github.com/woocommerce/woocommerce/pull/58012/commits/1e7a236edad3d05f8e5d8f1bc245d38a88b5840d.